docs/index.md

```diff
@@ -11,19 +11,31 @@
 
 # Tracee: Runtime Security and Forensics using eBPF
 
-Tracee is a Runtime Security and forensics tool for Linux. It uses Linux **eBPF
-technology** to trace your system and applications **at runtime**, and analyzes
-collected events in order to detect **suspicious behavioral patterns**. It is
-usually delivered as a docker container, but there are other ways you can use
-it (even create your own customized tracee container).
+Tracee is a runtime security and forensics tool for Linux based cloud deployments.
+It uses **eBPF** to trace the host OS and applications **at runtime**, and analyzes
+collected events in order to detect **suspicious behavioral patterns**. It can be
+run as a daemon-set in your kubernetes environment, but is flexible to be run for
+many purposes any Linux based hosts. It can be delivered via helm, as a docker
+container, or as a small set of static binaries.
+
+The goal of Tracee is to serve as an easy to use and effective solution for learning 
+when cloud native attacks occur in your environment. By leveraging Aqua's advanced 
+security research, performant eBPF based detection, and cloud native first
+approach Tracee makes runtime detection accesible, powerful, and effective.
 
 Watch a quick video demo of Tracee:
 
 [![Tracee Live Demo AND Q&A](./images/tracee_video_thumbnail.png)](https://youtu.be/x2_iF0KjPKs?t=2971)
 
 Check out the [Tracee video hub](https://info.aquasec.com/ebpf-runtime-security) for more videos.
 
-## Quickstart
+## Quickstart (Kubernetes)
+
+Tracee is designed to monitor hosts in kubernetes clusters. To see this in action check out the quickstart [here](./docs/getting-started/kubernetes-quickstart.md).
+
+## Quickstart (docker)
```

Comment:
Extract this to a `Docker - Quick Start`, which will be an option on the menu along with `Kubernetes - Quick Start` , in the overview part let's have the top text you changed + the section below the docker quick start where it says `Components`